setup.cfg

```diff
@@ -0,0 +1,45 @@
+# See https://packaging.python.org/en/latest/tutorials/packaging-projects
+
+[metadata]
+name = cbmc-viewer
+version = 2.10
+author = Mark R. Tuttle
+author_email = mrtuttle@amazon.com
+description = A CBMC viewer
```

Comment:
`Scans the output of CBMC and produces a browsable summary of the results.`

Reply:
Thanks.